Trappist/src/Promact.Trappist.Core/Controllers/TestsController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Models.Test;
+using Promact.Trappist.Repository.Tests;
+
+namespace Promact.Trappist.Core.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/tests")]
+    public class TestsController : Controller
+    {
+        private ITestsRepository _test;
+        private readonly TrappistDbContext _context;
+        public TestsController(ITestsRepository test)
+        {
+            _test = test;
+            //_context = context;
+        }
+        [HttpGet]
+        public string test1()
```

Comment:
Naming Convention of method is invalid. And Name should not like test 1, test 2 etc.